src/roman_datamodels/dqflags.py

```diff
@@ -76,5 +76,6 @@ class group(np.uint32, Enum):
     JUMP_DET   = pixel.JUMP_DET
     DROPOUT    = pixel.DROPOUT
     AD_FLOOR   = pixel.AD_FLOOR
+    WFI18_TRANSIENT = 2**7  # Affected by the WFI18 transient anomaly
```

Comment:
Are these going to propagated to the pixel level flags? If so We should use one of the reserved flags.